Can not find license keys associated with account

### Bug description

The container will fail to fetch a license from the FoundryVTT site.  My guess is that the license web page was modified sufficiently to break the scraper.

See: 
- https://discord.com/channels/170995199584108546/675738274572271656/1021993056754470995
- https://github.com/felddy/foundryvtt-docker/actions/runs/3096933113/jobs/5013959656#step:14:170


### Steps to reproduce

- Start a container
- Note the log message `Could not find any license keys associated with account <username>`

### Expected behavior

Expected that a license would be fetched.

### Container metadata

```shell
com.foundryvtt.version = "10.286"
org.opencontainers.image.authors = "markf+github@geekpad.com"
org.opencontainers.image.created = "2022-09-20T21:36:07.057Z"
org.opencontainers.image.description = "An easy-to-deploy Dockerized Foundry Virtual Tabletop server."
org.opencontainers.image.licenses = "MIT"
org.opencontainers.image.revision = "7144434df48d7551e8b64b0664ea597fb0b0d68f"
org.opencontainers.image.source = "https://github.com/felddy/foundryvtt-docker"
org.opencontainers.image.title = "foundryvtt-docker"
org.opencontainers.image.url = "https://github.com/felddy/foundryvtt-docker"
org.opencontainers.image.vendor = "Geekpad"
org.opencontainers.image.version = "10.286.0"
```


### Relevant log output

```console
E           Entrypoint | 2022-09-21 07:19:31 | [info] Installing Foundry Virtual Tabletop 10.286
160
E           Entrypoint | 2022-09-21 07:19:35 | [debug] Installation completed.
161
E           Entrypoint | 2022-09-21 07:19:35 | [info] Preserving release archive file in cache.
162
E           Entrypoint | 2022-09-21 07:19:35 | [debug] Patching GUI update and configuration messages.
163
E           Entrypoint | 2022-09-21 07:19:35 | [info] Installation not yet licensed.
164
E           Entrypoint | 2022-09-21 07:19:35 | [debug] Ensuring /data/Config directory exists.
165
E           Entrypoint | 2022-09-21 07:19:35 | [info] Attempting to fetch license key from authenticated account.
166
E           License | 2022-09-21 07:19:35 | [debug] Reading cookies from: cookiejar.json
167
E           License | 2022-09-21 07:19:35 | [info] Fetching licenses.
168
E           License | 2022-09-21 07:19:35 | [debug] Fetching: https://foundryvtt.com/community/***/licenses
169
E           License | 2022-09-21 07:19:35 | [error] Could not find any license keys associated with account ***
170
```


### Code of Conduct

- [X] I agree to follow this project's Code of Conduct

--- a/src/get_license.ts
+++ b/src/get_license.ts
@@ -76,7 +76,7 @@ async function fetchLicenses(username: string): Promise<string[]> {
   const body = await response.text();
   const $ = await cheerio.load(body);
 
-  const licenses: string[] = $("pre.license-key code")
+  const licenses: string[] = $("pre.key-code code")
     .map(function (this: cheerio.Element) {
       return $(this).text().replace(/-/g, ""); // remove dashes
     })

